clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -345,16 +513,28 @@ public String toString() {
         }
     }
 
-    private static class ConfigKey {
+    public static class ConfigKey {
         public final String name;
         public final Type type;
         public final String documentation;
         public final Object defaultValue;
         public final Validator validator;
         public final Importance importance;
+        public final String group;
+        public final int orderInGroup;
+        public final Width width;
+        public final String displayName;
+        public final List<String> dependents;
+        public final Recommender recommender;
 
         public ConfigKey(String name, Type type, Object defaultValue, Validator validator, Importance importance, String documentation) {
-            super();
+            this(name, type, defaultValue, validator, importance, documentation, null, -1, Width.NONE, name, null, null);
```

Comment:
Can we just change the one place this older, shorthand version is used and drop this constructor?